### PR TITLE
Add EXTCODEHASH opcode implementation

### PR DESCRIPTION
**Pull request by jwasinger**

### CODE REVIEW

File: lib/opFns.js [hunk 1]
```diff
@@ -288,6 +288,18 @@ module.exports = {
       cb(null)
     })
   },
+  EXTCODEHASH: function (address, runState, cb) {
+    var stateManager = runState.stateManager
```
Comment (axic) [inline lib/opFns.js]: Need to add check if it is on constantinople, otherwise throw.

Comment (jwasinger) [reply lib/opFns.js]: 👍 (Note to self: check that there are state tests that cover this edge case)

**Comment by axic** [comment]
I'm not sure what is the proper process: are there any state tests merged for this yet? If not, should we merge it and then fix once the tests are added? I think I vote for merging and lets assume the tests will be added as they are required for consensus.

**Comment by jwasinger** [comment]
Sounds good to me.  I'm working on state tests for `EXTCODEHASH`.

**Comment by jwasinger** [comment]
Moving this to: https://github.com/ethereumjs/ethereumjs-vm/pull/326
